Adding a folder housing the FNet input/target normalization optimization experiment with 1 notebook + scripts and outputs (#2)
WayScience/pediatric_cancer_atlas_analysis: 1.model_optimization/1.1.optimize_io_normalization_method/README.md

> # This folder contains code/notebook for running optimziation trials on the normalization method used for input and target images working with FNet models.

Reviewer: Is this another way to state this?
```suggestion
# This folder contains code for optimizing FNet models after normalizing input and target images.
```

Author: I might be misunderstanding you but the suggested change does not reflect what the code is trying to do here. This analysis is about finding the optimal combination of input and target normalization method (which is done with optuna some what awkwardly because instead of sampling a numerical hyperparameter we are actually sampling 3 normalizations * 2 positions (which are the input and target)). The actual FNet hyper-parameter optimization will be in another PR. I'll try to think of a better description for this one.

> This optimization analysis (and other optimizations in general) is dependent on output from running the notebooks under 0.data_preprocessing

Reviewer: ```suggestion
Our optimizations are primarily dependent on the data processed in `0.data_preprocessing`.
```

> For the analysis in this folder to work, the config.yml file at the repo home will need to be configured with the correct pathes to the pediatric_cancer_atlas_profiling repo (`pediatric_cancer_atlas_profiling_path`), the location where the single cell level features parquet files (`sc_features_path`) as well as the location where the virtual_stain_flow_software (dev-0.1) branch clone lives (`software_path`) as the software is currently in active development and is not installable. 

Reviewer: ```suggestion
For the analysis in this folder to work, the config.yml file at the repo home will need to be configured with the correct paths to the pediatric_cancer_atlas_profiling repo (`pediatric_cancer_atlas_profiling_path`), the location where the single cell level features parquet files (`sc_features_path`) as well as the location where the virtual_stain_flow_software (dev-0.1) branch clone lives (`software_path`) as the software is currently in active development and is not installable. 
```

Reviewer: Consider reducing this to multiple sentences

> ## The FNet input/output(target) normalization optimization trails can be run by running the following notebook:

Reviewer: ```suggestion
## The FNet input and output (target) normalization optimization can be performed by running the following notebook:
```

1.1.optimize_io_normalization_method/1.1.1.optimize_fnet_io_norm.ipynb

## ALternatively, run
```bash

> # Ensure config.yml at repo root is properly configured

Reviewer: ```suggestion
# Ensure `config.yml` at the root of the repository is configured
```
